src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -55,7 +55,9 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction BooleanN = _library.Add(new BooleanNFunction());
         public static readonly TexlFunction BooleanN_T = _library.Add(new BooleanNFunction_T());
         public static readonly TexlFunction BooleanW = _library.Add(new BooleanWFunction());
-        public static readonly TexlFunction BooleanW_T = _library.Add(new BooleanWFunction_T());
+        public static readonly TexlFunction BooleanW_T = _library.Add(new BooleanWFunction_T());

```

Comment:
IIRC this will immediately expose them in PA Client. Do you have the corresponding JS implementations ready?

Reply:
Done.  By making this change, the new functions don't appear in signature help, but I assume that is a testing thing not pulling in the feature gated functions.